user-guide/Troubleshooting/Procedures/Web_Issues_Support_Assistant.md

```diff
@@ -8,9 +8,9 @@ The DataMiner Web Support Assistant is a **Chrome extension** designed to assist
 
 ## Installing the extension
 
-To add the DataMiner Web Support Assistant extension to your Google Chrome browser:
+To add the DataMiner Web Support Assistant extension to your Chromium browser:
 
-1. Open [this link](https://chromewebstore.google.com/detail/dataminer-web-support-ass/nofmcbgpolhjblmafpfbffjnganhapge) and click *Add to Chrome*.
+1. Open [this link](https://chromewebstore.google.com/detail/dataminer-web-support-ass/nofmcbgpolhjblmafpfbffjnganhapge) and click *Download*.
```

Comment:
I've just tested this, and in Google Chrome (which I suspect most people will be using), this button is "Add to Chrome", not "Download", so I think this replacement could cause some confusion. Perhaps we could rephrase this so it works regardless of which Chromium browser exactly is used?
```suggestion
1. Open [this link](https://chromewebstore.google.com/detail/dataminer-web-support-ass/nofmcbgpolhjblmafpfbffjnganhapge) and click the button to deploy the extension.
```